vocabulary/activitystreams2.owl

```diff
@@ -518,6 +518,16 @@ as:published a owl:DatatypeProperty ,
   rdfs:range xsd:dateTime ;
   rdfs:domain as:Object .
 
+# TODO BUG The spec defines this as 'Object | Link | xsd:dateTime | xsd:boolean' 
+# which is not OWL-compatible (both ObjectProperty and DatatypeProperty).
+as:closed a owl:DatatypeProperty ,
+      owl:FunctionalProperty ;
+  rdfs:label "closed"@en ;
+  rdfs:comment "Indicates that a question has been closed, and answers are no longer accepted."@en ;
+  rdfs:range xsd:dateTime ;
+  rdfs:range xsd:boolean ;
+  rdfs:domain as:Object .
```

Comment:
In the meantime, what's true and safe is something like:

```suggestion
as:closed a rdf:Property ;
  rdfs:label "closed"@en ;
  rdfs:comment "Indicates that a question has been closed, and answers are no longer accepted."@en .
```

Reply:
That issue is related to the inconsistency between the normative JSON-LD context and the normative HTML documentation. My goal is to decrease the discrepancies between the AS2 OWL ontology, the normative JSON-LD context, and the normative HTML document as much as possible (and to the extent it makes sense in an OWL context). To that end, I should remove the `rdfs:range xsd:boolean` to be consistent with the normative context. Also, the `rdfs:domain` is not accurate. I will change that to `as:Question`.